topdown/crypto_test.go

```diff
@@ -112,100 +187,110 @@ nwy7dzejHmQUcZ/aUNbc4VTbiv15ESk=
 }
 
 func TestParseRSAPrivateKey(t *testing.T) {
```

Comment:
I think we can roll `TestParseRSAPrivateKey` into `TestParsePrivateKey`?

I also think it'd be nice to lay it out as one big table test: https://golang.cafe/blog/golang-table-test-example